### ISSUE TITLE
Scheduler 'Until' - End on date - sets to the initial date at a repeat period change

### ISSUE THREAD

**Issue by netscasy** [ExactTarget/fuelux#1755]
Scheduler repeat 'Until' (End on date) presets to the current date and remains so even after it is set to another future date.
However, if the 'End on date' is set and then the 'repeat period' is changed, say from 'daily' to 'weekly', the 'End on date' previously set for 'daily', will be set as the new 'Until' date and will be reflected in the final 'recurrencePattern'. Any changes in the 'End on date' with the same 'repeat period' selected, will not be reflected in the 'recurrencePattern'. 

In short the 'End on' date always defaults to the value at the time of a 'repeat period' change event and do not change unless a a different 'repeat period' is selected.
